words2num/lang_EN_US.py

```diff
@@ -113,7 +113,8 @@ def f_ret(self, _):
             ('X', 'F'): f_ret,     # 9000
             ('Z', 'F'): f_ret,     # 0
             ('A', 'H'): f_mul_hundred,     # 100
-            ('A', 'X'): f_mul      # 1000
+            ('A', 'X'): f_mul,      # 1000
+            ('A', 'F'): f_ret,      # 1
```

Comment:
What does this need to be added for?

Reply:
Since I'm stripping off "multipliers" from the end of the input word, we sometimes run into the situation where the only remaining word is "A", and words2num wasn't previously able to handle that.